TutorialApplication/MainViewController.m

```diff
@@ -26,32 +30,50 @@ - (void)viewDidLoad
 {
     [super viewDidLoad];
     
-    Employee *emp1 = [[Employee alloc] initWithName:@"Stepka" lastName:@"Pupkin" salary:3000];
-    
-    self.organization = [[Organization alloc] initWithName:@"TheBestOrganization"];
+    self.organization = [DatabaseController requestResultsForPredicate:nil sortDescriptors:nil entity:@"FFOrganization"].firstObject;
     
-    [self.organization addEmployee:emp1];
-    [self.organization addEmployeeWithName:@"Alex Yegorov"];
-    [self.organization addEmployeeWithName:@"Vova Kynovskiy"];
-    [self.organization addEmployeeWithName:@"Misha Davidayn"];
+    if (self.organization == nil)
+    {
+        self.organization = [NSEntityDescription insertNewObjectForEntityForName:@"FFOrganization" inManagedObjectContext:[DatabaseController sharedInstance].context];
+        self.organization.name = @"Best";
+        
+        FFEmployee *entityNameObj = [NSEntityDescription insertNewObjectForEntityForName:@"FFEmployee" inManagedObjectContext:[DatabaseController sharedInstance].context];
+        entityNameObj.firstName = @"Eva";
+        entityNameObj.lastName = @"Green";
+        entityNameObj.salary = 15600;
+        entityNameObj.fullName = @"Eva Green";
+        
+        [self.organization addEmployeesObject:entityNameObj];
+        
+        [self.organization addEmployeesObject:[self.organization addEmployeeWithName:@"Miss Smith"]];
```

Comment:
[self.organization addEmployeeWithName:@"Miss Smith"] should not return an employee to be added, you should modify this method, so it returns void and automatically adds an employee to the organization.